### PR TITLE
Gitdiscovery

### PR DESCRIPTION
**Pull request by bastell**
discovery of git repos as well as linking to manifest files

### CODE REVIEW

File: src/pyatsimagebuilder/utils.py [hunk 1]
```diff
@@ -67,6 +68,39 @@ def ftp_retrieve(host, from_path, to_path, port=None, secure=False):
     ftp.close()
 
 
+def git_info(path, repo=None):
```
Comment (tahigash) [inline src/pyatsimagebuilder/utils.py:71]: not fully understand what this is for. maybe better to update README with this change and also doc relating to manifest?

**Review by dwapstra — APPROVED** [review]

File: src/pyatsimagebuilder/utils.py (hunk 1 above)
Comment (bastell) [reply src/pyatsimagebuilder/utils.py:71]: I've added the various discovery files with a brief description to the README.

File: README.md [hunk 2]
```diff
@@ -467,26 +467,36 @@ pyATS Docker images created using this package features the following directory
 structure:
 
 ```text
-/venv
+/pyats
```
Comment (tahigash) [inline README.md:470]: why this was changed? I'm worrying about impact where we need to update docs and training materials.

File: README.md [hunk 3]
```diff
@@ -467,26 +467,36 @@ pyATS Docker images created using this package features the following directory
...
     Directory where the Python virtual environment is created. All Python
     packages (including pyATS) specified in the build YAML file are installed
-    into here.
+    into here. Additionally acts as the workspace, where all files and
+    repositories specified in the YAML build file gets copied to. Set as the
+    Docker working directory.
 
-/workspace
```
Comment (tahigash) [inline README.md:475]: same with above. no more `/workspace`? I assume we need to update docs and etc. when it comes to the impact where we need to update, I'm feeling not good idea to change this... or will you take care of those and already knew not impacted...?

Comment (bastell) [reply README.md:475]: `/workspace` and `/venv` were merged to become `/pyats` many months ago, that's not the purpose of this PR. The readme has just been updated to reflect this now. I'll go double check that all the paths are correct in the readme. This PR only adds the `repos.json` file and modifies the `manifest.json` file, which are records of available git repos and manifest files, but do not impact the actual execution of the final docker image.

**Review by tahigash — APPROVED** [review]

**Pull request merged**